library/L1_Peripheral/lpc40xx/test/eeprom_test.cpp

```diff
@@ -0,0 +1,92 @@
+/// This is the eeprom.cpp test file
+
+#include "L1_Peripheral/lpc40xx/eeprom.hpp"
+#include "L4_Testing/testing_frameworks.hpp"
+
+namespace sjsu::lpc40xx
+{
+EMIT_ALL_METHODS(Eeprom);
+
+TEST_CASE("Testing EEPROM", "[Eeprom]")
+{
+    // Simulate local version of LPC_EEPROM
+    LPC_EEPROM_TypeDef local_eeprom;
+
+    // Clear memory locations and assign eeprom_register to local variable
+    memset(&local_eeprom, 0, sizeof(local_eeprom));
+    Eeprom::eeprom_register = &local_eeprom;
+
+    // Set mock for sjsu::SystemController
+    constexpr uint32_t kDummySystemClockFrequency = 48'000'000;
+    Mock<sjsu::SystemController> mock_system_controller;
+    When(Method(mock_system_controller, GetSystemFrequency))
+        .AlwaysReturn(kDummySystemClockFrequency);
+
+    // Creating test Eeprom object
+    Eeprom test_eeprom(mock_system_controller.get());
+
+    // Write/Read tests only check one 32-bit transfer, so size is 4 bytes
+    constexpr size_t kPayloadSize = 4;
+
+    SECTION("Initialization")
+    {
+        test_eeprom.Initialize();
+
+        // Assuming clock speed of 48 Mhz
+        // Wait State Register Values should be 2, 3, and 1
+        constexpr uint32_t kWaitStateValues = 0b00000010'00000011'00000001;
+        // Clock divider should be set to 128
+        constexpr uint32_t kClockDivider    = 0b10000000;

```

Comment:
```suggestion
        constexpr uint32_t kClockDivider    = 128;
```
Just making the expected clock divider 128 makes this more readable, rather than having it in binary here. It also removes the need for the comment below.